Timer.DAL/Timer.DAL/UnitOfWork/IUnitOfWork.cs

```diff
@@ -1,11 +1,12 @@
-﻿using gtdtimer.Timer.DAL.Entities;
+﻿using Microsoft.AspNet.Identity;
+using gtdtimer.Timer.DAL.Entities;
 using gtdtimer.Timer.DAL.Repositories;
 
 namespace gtdtimer.Timer.DAL.UnitOfWork
 {
     public interface IUnitOfWork
     {
-        IRepository<User> Users { get; set; }
+       ApplicationUserManager Users { get; set; }
```

Comment:
some interface would be better

Reply:
I think so but ApplicationUserManager inherit from UserManager, but UserManager doesn't inherit from some interfaces or classes
![2018-12-14_00-04-06](https://user-images.githubusercontent.com/22987347/49970403-d2cf0e00-ff33-11e8-8759-a8d8de00e353.png)